src/Database/TypeInterface.php

```diff
@@ -61,18 +61,19 @@ public function marshal($value);
 
     /**
      * Returns the base type name that this class is inheriting.
-     * This is useful when extending base type for adding extra functionality
+     * 
```

Comment:
Odd, when I open the file either in my IDE or on GH (raw), I don't see the whitespace.